internal/providers/terraform/aws/registry.go

```diff
@@ -180,6 +181,12 @@ var FreeResources []string = []string{
 	"aws_s3_bucket_policy",
 	"aws_s3_bucket_public_access_block",
 
+	// AWS KMS
+	"aws_kms_alias",
+	"aws_kms_ciphertext",
+	"aws_kms_external_key",
```

Comment:
The `aws_kms_external_key` needs the `Customer master key` cost only (no requests). Can you possibly add this as a resource and put the `Customer master key` cost in a function so `aws_kms_external_key` calls that function? See the `elb.go` file for an example of how this was done elsewhere.